packages/babel-core/src/config/files/configuration.js

```diff
@@ -5,45 +5,35 @@ import path from "path";
 import fs from "fs";
 import json5 from "json5";
 import resolve from "resolve";
-import { makeStrongCache, type CacheConfigurator } from "../caching";
+import {
+  makeStrongCache,
+  makeWeakCache,
+  type CacheConfigurator,
+} from "../caching";
 import makeAPI from "../helpers/config-api";
+import { makeStaticFileCache } from "./utils";
+import type { FilePackageData, RelativeConfig, ConfigFile } from "./types";
 
 const debug = buildDebug("babel:config:loading:files:configuration");
 
-export type ConfigFile = {
-  filepath: string,
-  dirname: string,
-  options: {},
-};
-
-export type IgnoreFile = {
-  filepath: string,
-  dirname: string,
-  ignore: Array<string>,
-};
-
-export type RelativeConfig = {
-  config: ConfigFile | null,
-  ignore: IgnoreFile | null,
-};
+const BABEL_CONFIG_FS_FILENAME = "babel.config.js";
```

Comment:
why the `FS` in the constant's name? is it for file system? seems weird that only this one has anything other than default name + filename suffix in its identifier

Reply:
That's definitely a typo of `JS` hahah

Reply:
Let's fix that before merging.